### ISSUE TITLE
Run tests on Appveyor to debug rplugin on Windows

### ISSUE THREAD

**Issue by janlazo** [neovim/node-client#47]
After work done on https://github.com/neovim/neovim/pull/7706, https://github.com/neovim/neovim/pull/7734, https://github.com/neovim/neovim/pull/7738, I can't make a simple echo class with an `EchoHello` command work on Windows. The APIs are fine because testing on them directly, bypassing the host and `rplugin/node`,  works on all OS consistently. `:checkhealth` and `:UpdateRemotePlugins` are ok now (on Windows) and I see `rplugin.vim` with `EchoHello` registered from nodejs plugin I made it.

That leaves msgpack and the `RPC` api from both sides which I don't know how to debug/fix yet.
To get more debug info, the tests should run on Appveyor as well. We can reuse the same configuration from neovim repo.

**Comment by janlazo** [comment]
NVIM_NODE_LOG_FILE

```
2017-12-20T18:15:31.532Z - debug: host.start

2017-12-20T18:16:06.580Z - debug: host.start

2017-12-20T18:16:06.612Z - info: handleRequest:  poll

2017-12-20T18:16:06.626Z - debug: request received:  poll

2017-12-20T18:16:06.629Z - info: handleNotification:  C:/Users/JanEdmund/AppData/Local/nvim/rplugin/node/hello.js:command:EchoHello

2017-12-20T18:16:06.630Z - debug: host.handlePlugin:  C:/Users/JanEdmund/AppData/Local/nvim/rplugin/node/hello.js:command:EchoHello

2017-12-20T18:16:06.766Z - info: [C] Error loading child ChildPlugin C

2017-12-20T18:16:06.789Z - info: [C] { Error: Cannot find module 'C'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at Object.sandboxRequire [as require] (C:\Users\JanEdmund\lib\node6\node_modules\neovim\lib\host\factory.js:79:46)
    at createPlugin (C:\Users\JanEdmund\lib\node6\node_modules\neovim\lib\host\factory.js:105:39)
    at Object.loadPlugin (C:\Users\JanEdmund\lib\node6\node_modules\neovim\lib\host\factory.js:139:16)
    at Host.getPlugin (C:\Users\JanEdmund\lib\node6\node_modules\neovim\lib\host\index.js:23:59)
    at Host.<anonymous> (C:\Users\JanEdmund\lib\node6\node_modules\neovim\lib\host\index.js:39:33)
    at next (native)
    at C:\Users\JanEdmund\lib\node6\node_modules\neovim\lib\host\index.js:7:71 code: 'MODULE_NOT_FOUND' }
```

**Comment by janlazo** [comment]
With #48, https://github.com/neovim/node-client/issues/43#issuecomment-351799952 works with command registration such that I can run a custom command to do `echomsg "Hello"`. It still doesn't work if using the host (bin/cli.js). 

```
2017-12-21T07:54:12.319Z - debug: host.start
2017-12-21T07:54:12.351Z - info: handleRequest:  poll
2017-12-21T07:54:12.364Z - debug: request received:  poll
2017-12-21T07:54:12.368Z - info: handleNotification:  C:/Users/JanEdmund/AppData/Local/nvim/rplugin/node/hello.js:command:EchoHello
2017-12-21T07:54:12.369Z - debug: host.handlePlugin:  C:/Users/JanEdmund/AppData/Local/nvim/rplugin/node/hello.js:command:EchoHello
2017-12-21T07:54:12.400Z - debug: getPlugin.shouldCache false
2017-12-21T07:54:12.402Z - info: [hello.js] Run echoHello()
2017-12-21T07:54:12.403Z - debug: request -> neovim.api.nvim_command
2017-12-21T07:54:12.406Z - debug: response -> neovim.api.nvim_command: null
```

**Comment by janlazo** [comment]
Can there be a patch release soon for Windows users?

**Comment by rhysd** [comment]
I don't have permission to publish. @billyvg, could you do that?

**Comment by billyvg** [comment]
Done, published 3.5.2

**Comment by rhysd** [comment]
Thank you.
